pkg/api/types.go

```diff
@@ -527,6 +530,32 @@ type TestStepConfiguration struct {
 	OpenshiftInstallerCustomTestImageClusterTestConfiguration *OpenshiftInstallerCustomTestImageClusterTestConfiguration `json:"openshift_installer_custom_test_image,omitempty"`
 }
 
+// Cloud is the name of a cloud provider, e.g., aws cluster topology, etc.
+type Cloud string
+
+const (
+	CloudAWS Cloud = "aws"
+)
+
+// ClusterClaim claims an OpenShift cluster for the job.
+type ClusterClaim struct {
+	// Product is the name of the product being released.
+	// Defaults to ocp.
+	Product ReleaseProduct `json:"product,omitempty"`
+	// Version is the version of the product
+	Version string `json:"version,omitempty"`
```

Comment:
The non-optional fields should probablky not be omitempty?

Reply:
Updated.